content.tex

```diff
@@ -52,7 +52,7 @@ \subsubsection{Pre-replication survey: available materials}
 I managed to find a backup on an old hard drive. In this archive, 
 there is a folder named \texttt{code} with some 30 versions,
 consistently named in a pattern \texttt{mon-date}. (Source control? Never
-heard of him.) Sadly, the year did not make it into the naming scheme, and the
+heard of it.) Sadly, the year did not make it into the naming scheme, and the
```

Comment:
Huh. As far as I can tell, this was meant to be modeled after a conversation from the Hobbit: "Gandalf? Never heard of him". 
So a reproducibility take-away, duly noted: jokes which where meant to be funny when written look just cheesy after a couple of years. 
Thanks!